packages/ui/theme/common.tsx

```diff
@@ -63,6 +63,8 @@ const colors = {
 		coolGray: '#d9d9d9',
 		red: '#C05C4A',
 		pink: '#D4A1BA',
+		lightPink: '#F2E1E9',
+		lightGreen: '#E4F9EA',
```

Comment:
I chose the light green based on this https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors

This was not in the design, so not sure if we want to use this instead of the lightPink for all states.